Classes/Domain/Repository/FolderRepository.php

```diff
@@ -33,7 +34,7 @@ public function getFolder(\TYPO3\CMS\Core\Resource\Folder $folder): ?Folder
             ->where(
                 $db->expr()->eq(
                     'folder',
-                    $db->createNamedParameter($folder->getName())
+                    $db->createNamedParameter($folder->getIdentifier())
```

Comment:
This is not explained - neither in the commit message, PR message nor as comment in the code.

What's the reason for this change ? Is this safe for v11 AND v12 ? Is this a "bugfix" because being wrong beforehand - or is this related to a core change ?

Can you enlight me and others on that point ? Please :smile:

Reply:
getName only returned the name, but identifier returns full path. As name could be multiple, identifier is unique inside a storage. This is failsafe for getting the right storage folder.